pantheon-bundle/src/main/java/com/redhat/pantheon/sling/ServiceResourceResolverProvider.java

```diff
@@ -1,5 +1,6 @@
 package com.redhat.pantheon.sling;
 
+import com.redhat.pantheon.model.api.SlingModel;
```

Comment:
Why is this new import needed? the only change in this class other than the import are comments.

Reply:
You are correct sir :smile: